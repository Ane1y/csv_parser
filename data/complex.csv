  ,A,    V,   Vell,         M
1, 5,  =Vell3,  =A1 * V1,   1
4, 4, =A1/A4, =A4+Vell1, 8
3, =A4-M4, 7, =A1, 3
