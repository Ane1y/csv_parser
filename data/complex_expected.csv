,A,V,Vell,M
1,5,5,25,1
4,4,1,29,8
3,-4,7,5,3
